app/models/author.rb

```diff
@@ -1,7 +1,25 @@
+#
+# frozen_string_literal: true
+
 class Author < ApplicationRecord
   has_many :publications
   has_many :books, through: :publications
   has_many :comments, as: :commentable
-  validates :name, presence: true
+  validates :name, presence: true, format: { with: /\A[a-zA-Z]+\z/,
```

Comment:
please, format this in a more readable way.

somthing like: 
  validates(
    :name,
   etc.
)